lib/Doctrine/DBAL/Connection.php

```diff
@@ -1971,6 +1971,19 @@ public function getSchemaManager()
         return $this->_schemaManager;
     }
 
+    /**
+     * Creates the SchemaManager that can be used to inspect or change
+     * the database schema through the connection.
+     *
+     * This method is an improvement in the upgrade path to DBAL 3.
+     *
+     * @return AbstractSchemaManager
+     */
+    public function createSchemaManager()
```

Comment:
Is there a reason not to use a native return type declaration here?
```suggestion
    public function createSchemaManager(): AbstractSchemaManager
```

Reply:
I've just copied `getSchemaManager()` from the same file instead of copying `createSchemaManager()` from 3.3.x... :man_shrugging:

Reply:
Fixed now.